docs/contributing/coding-style.md

```diff
@@ -776,6 +778,34 @@ tests := map[string]func(t *testing.T) test {
     }
     ```
 
+#### Inputs
+
+1. test case name
+
+    Test case name should be readable, meaningful and understandable easily. 
+    If you create a new test, the test name should be named based on the below naming templates.
+
+    - Success cases:
+        - Start with `success` or `{verb} success` or `success {verb}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `set success when the value is default value`
+    - Fail/Error cases:
+        - Start with `fail` or `{verb} fail` or `fail {verb}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `fail option setting when value is invalid value(string)`
+    - Return cases:
+        - If the test case do not match with `Success` or `Fails`, please use `Return` pattern.
+        - Start with `Returns {Object}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `return invalid error when the input is invalid`
+
+1. testing arguments
+
+    Input arguments for testing should be a meaningful value.
+    We should test with more realistic value as user use, to produce more realistic testing result.
```

Comment:
```suggestion
    Input arguments for testing should be a meaningful value.
    We should test with more realistic value as user use, to produce a more realistic testing result.
```

Reply:
```suggestion
    Input arguments for testing should be a meaningful value.
    We should test with more realistic value as user use, to produce more realistic testing results.
```

Reply:
thank you for your suggestion. 🙇 
It looks good to me.
I think your suggestion is good.

Reply:
I think `testing result` is correct